eng/dependabot/Packages.props

```diff
@@ -21,7 +21,7 @@
     <PackageReference Update="System.Private.Uri" Version="4.3.2" />
     <PackageReference Update="System.Runtime" Version="4.3.1" />
     <PackageReference Update="System.Security.Cryptography.Pkcs" Version="6.0.1" />
```

Comment:
@YuliiaKovalova looks like we need 6.0.4 of the Pkcs file as well. Not sure why that didn't show up in my initial pass.